src/components/editor/leftNav/RelationshipsDisplay.jsx

```diff
@@ -8,14 +8,12 @@ import { selectRelationships } from "selectors/relationships"
 import { getSearchResultsByUris } from "sinopiaSearch"
 import { addError } from "actions/errors"
 import RelationshipRow from "./RelationshipRow"
+import useAlerts from "hooks/useAlerts"
 import _ from "lodash"
 
-const RelationshipsDisplay = ({
-  resourceKey,
-  errorKey,
-  displayActions = true,
-}) => {
+const RelationshipsDisplay = ({ resourceKey, displayActions = true }) => {
```

Comment:
You'll see lots of changes like this -- using error key from context instead of passing as property.